root/usr/local/src/osticket/setup/install.php

```diff
@@ -82,7 +83,7 @@ function convertStrToBool($varName, $default) {
 $mailConfig = str_replace('%SMTP_TLS_CERTS%', $vars['smtp_tls_certs'], $mailConfig);
 
 $mailConfig = str_replace('%SMTP_TLS%', boolToOnOff(convertStrToBool('smtp_tls',true)), $mailConfig);
-$mailConfig = str_replace('%SMTP_AUTH%', boolToOnOff($vars['smtp_user'] != ''), $mailConfig);
+$mailConfig = str_replace('%SMTP_AUTH%', !empty($vars['smtp_auth']) ? $vars['smtp_auth'] : boolToOnOff(getenv("SMTP_USER") != ''), $mailConfig);
```

Comment:
Why the change from `$vars['smtp_user']` to `getenv("SMTP_USER")` in `boolToOnOff` call?